domxpath/fn-lang.html

```diff
@@ -34,14 +34,5 @@
 // if there is some suffix starting with - such that the attribute value is
 // equal to the argument ignoring that suffix of the attribute value
 testFirstChild('lang("ja")', '<root xml:lang="ja-jp"><unmatch xml:lang="ja_JP"/></root>');
-
-// U+212A should match to ASCII 'k'.
-// XPath 1.0 says:
-// ... such that the attribute value is equal to the argument ignoring that suffix
-// of the attribute value and ignoring case.
-// XPath 3.1 says:
-// ... true if and only if, based on a caseless default match as specified in
-// section 3.13 of The Unicode Standard,
-testFirstChild('lang("ko")', '<root><match xml:lang="&#x212A;o"/></root>');
```

Comment:
I would change this to use the `<unmatch>` element instead. I don't think we need to separate out this test.

I'd also remove the 3.1 reference from help at the top of this file and clarify that 3.1 is not to be implemented as per https://github.com/whatwg/dom/issues/1199.

Reply:
Updated. :-)